src/previewbsa.cpp

```diff
@@ -159,7 +159,7 @@ QWidget* PreviewBsa::genBsaPreview(const QString& fileName, const QSize&)
   layout->addWidget(infoLabel);
 
   QTreeView* view            = new QTreeView();
-  SimpleFileTreeModel* model = new SimpleFileTreeModel(m_Files);
```

Comment:
Passing the view to the model is kind of an anti-pattern, is that really necessary? What is something you tried for ownership?